testsuite/features/step_definitions/command_steps.rb

```diff
@@ -1456,29 +1456,33 @@
 end
 
 When(/^I change the server's short hostname from hosts and hostname files$/) do
-  old_hostname = get_target('server').hostname
+  server_node = get_target('server')
```

Comment:
Is this enough to get the server FQDN/hostname?

Reply:
No, is not.
The next line will retrieve it.